src/shared/components/hero-section/index.module.css

```diff
@@ -112,32 +105,30 @@
             line-height: var(--line-height-large);
         }
 
-        & p {
-            font-size: var(--typography-text-size);
-            font-weight: var(--typography-text-weight-light);
-            text-align: center;
-            line-height: var(--line-height-medium);
+        & .textDesc {
+            max-width: 70%;
+
+            & p {
+                font-size: var(--typography-text-size);
+                font-weight: var(--typography-text-weight-light);
+                text-align: center;
+                line-height: var(--line-height-medium);
+            }
         }
 
         & .infoContainer {
             width: 100%;
             margin-top: 2%;
             padding-bottom: calc(var(--bottom-half-height) * 2);
-            transform: scale(1);
             opacity: 1;
             color: var(--color-light-blue);
             font-family: var(--typography-roboto-text-font);
             font-size: var(--typography-text-size-mobile);
-            transition: opacity var(--info-transition-duration) ease, transform var(--info-transition-duration) ease;
-
-            &.error {
-                color: var(--color-error);
-            }
+            transition: opacity var(--info-transition-duration) ease;
 
             &.hidden {
-                transform: scale(0.9);
```

Comment:
Instead of doing all of this, why not simply `visibility: hidden`?

Reply:
To have the `fade in` effect when data is loaded.